asmcomp/asmpackager.ml

```diff
@@ -217,6 +217,9 @@ let build_package_cmx members cmxfile =
     else
       Clambda (get_approx ui)
   in
+  let ui_need_stdlib =
+    List.fold_left (fun acc unit -> acc || unit.ui_need_stdlib) false units
+  in
```

Comment:
```suggestion
  let ui_need_stdlib = List.exists (fun info -> info.ui_need_stdlib) units in
```

There's a similar pattern below for `ui_force_link`.

Reply:
I'm not sure what possessed me to write that originally! I blame the pandemic